ui/app/components/MethodTreeComponent/MethodTreeComponent.js

```diff
@@ -157,7 +160,7 @@ class MethodTreeComponent extends Component {
           .filter(filterPaths.bind(null, uniqueId));
         const nodeRender =
           <StackTreeElement
-            data-nodename={displayName}
```

Comment:
just check with Ankur, whether "data-nodename" was supposed to be used for something else.